README.md

```diff
@@ -279,10 +211,10 @@ I would like to thank...
 
 <!-- LICENSE -->
 
-## 📝 License <a name="license"></a>
+## 📝 License <a name="license" href="https://github.com/Munyabelden/helloSetup/blob/hello-microverse/MIT.md">Mit.md</a>
 
 This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
[OPTIONAL] I suggest you remove this section here. It is mean to assist you understand how to add an licence file. It makes this part look unprofessional.